modules/imgproc/src/color.cpp

```diff
@@ -5917,9 +5893,12 @@ static inline softfloat applyInvGamma(softfloat x)
                 pow(xd, softdouble::one()/gammaPower)*(softdouble::one()+gammaXshift) - gammaXshift);
 }
 
+static Mutex tablesInitMtx;
```

Comment:
It is not safe to use global static variable with constructor in general. Refer to 'static initialization order fiasco' problem. The most dangerous issues are:
- use before object constructor call
- use after object destroying